src/differentials/abstract_zero.jl

```diff
@@ -1,13 +1,13 @@
 """
     AbstractZero <: AbstractDifferential
 
-This is zero-like differential types.
-If a AD system encounter a propagator taking as input only subtypes of `AbstractZero` then
-it can stop performing any AD operations, as all propagator are linear functions, and thus
+This abstract type encompasses zero-like differential types.
+If an AD system encounters a propagator that takes as input only subtypes of `AbstractZero`,
+then it can stop performing AD operations. All propagators are linear functions, and thus
 the final result will be zero.
```

Comment:
Prefer at most one sentence per line, for nicer diffs
```suggestion
then it can stop performing AD operations.
All propagators are linear functions, and thus the final result will be zero.
```